client/src/components/BlockEditor/normalizeChange.js

```diff
@@ -0,0 +1,54 @@
+import { SPACER } from "../../lib/slate-pointers/exportedPointerSpacer";
+import { Change, TextNode } from './types';
+
+export function normalizeChangeWrtTextNodeSpacing(c: Change) {
```

Comment:
This is a pretty unwieldy function name. How about `normalizeExportSpacing`?

Reply:
Sure.

Reply:
https://github.com/oughtinc/mosaic/pull/145/commits/f8187e31d855b923d4aed229263bed31ab2cf8b4